File: experiments/runs/2026-08-17_06-39-23_claim-office-example-mapping_v3-basic-tdd_opus-4-8-no-thinking-3/src/rounding.ts
// Rounding "in the MHPCO's favor": premiums round up, payouts round down.
export function roundInFavor(amount: number, kind: 'premium' | 'payout'): number {
  return kind === 'premium' ? Math.ceil(amount) : Math.floor(amount);
}

File: experiments/runs/2026-09-22_01-57-46_claim-office-python-example-mapping_baseline-inline-tdd-v1-cc_opus-5-no-thinking/src/rounding.py
"""Rounding, always in the MHPCO's favour."""

import math


def round_in_favour_of_office(premium):
    """Premiums are rounded up."""
    return math.ceil(premium)


def round_in_favour_of_customer(payout):
    """Payouts are rounded down."""
    return math.floor(payout)
